package.json

```diff
@@ -19,6 +19,7 @@
   "peerDependencies": {
     "react": "^16.2.0",
     "react-dom": "^16.2.0",
+    "react-router": "^4.2.0",
```

Comment:
There are two areas in force that use React router, but the API between found and RR is basically the same. Will need to follow up and swap out.